src/medCore/gui/toolboxes/medRegistrationSelectorToolBox.cpp

```diff
@@ -207,14 +178,13 @@ void medRegistrationSelectorToolBox::changeCurrentToolBox(int index)
     }
 }
 
-
 //! Clears the toolbox.
 void medRegistrationSelectorToolBox::clear(void)
```

Comment:
The clear() method is actually such a mess, sometime it resets the tbx, sometimes it rm every widget, ...
I'll see what I can do